packages/core/src/indexing/postgres/migrations/1-create-model-table.ts

```diff
@@ -72,3 +72,30 @@ export async function createModelTable(
     )
   })
 }
+
+export async function createConfigTable(dataSource: Knex, tableName: string) {
+  switch (tableName) {
+    case 'ceramic_models':
+      await dataSource.schema.createTable(tableName, function (table) {
+        // create model indexing configuration table
+        table.bigIncrements('index_id')
+        table.string('model', 1024).notNullable()
+        table.boolean('is_indexed').notNullable().defaultTo(true)
+        table.dateTime('created_at').notNullable().defaultTo(dataSource.fn.now())
+        table.dateTime('updated_at').notNullable().defaultTo(dataSource.fn.now())
+        table.string('updated_by', 1024).notNullable()
```

Comment:
ugh, this is such a poor man's audit log.  Postgres already has built-in audit log capabilities for people who want it.  Maybe we should just have a `created_by` but not try to track updates ourselves?  Otherwise it's weird to me the idea that a user who makes a trivial change might override the data about the user who made the last substantially meaningful change.

Reply:
It's mainly meant to have some basic quick audit capability for an Admin to follow up e.g. if a model was deactivated by a developer by mistake for follow ups. 

Yes, there is the Postgres audit log capability (`pgAudit`) but that needs to be separately activated and is specific to Postgres again (right now that big of an issue really). As far as I remember that would also rely on different auth connections to differentiate users though. Since developers connect via the node connection string, I think the audit log would be less useful than to set explicit access via the DID as initially proposed. Overall, the last update on the status (`is_indexed`) is really the only interesting and relevant on in combination with the `updated_at` which is an indication of when models stopped being indexed -> this potentially could be useful for historical sync as well to check against the mid table history.

Reply:
Ugh, while I see the point of that, it is better to implement an audit log separately, IMO.

Reply:
@stbrody @ukstv Let's jump on a sync on this one. We can implement a separate audit log but would need to extend the scope of the current spec then and decide if its part of this or next release milestone.

Reply:
Yup. Do we even need an audit log for January release?

Reply:
@stbrody I am sold we better keep it.

Reply:
Going forward with the current implementation but filed a ticket to extend history: https://linear.app/3boxlabs/issue/CDB-1872/spec-log-history-for-ceramic-models-table